frontend/src/pages/WelcomePage/WelcomePage.module.css

```diff
@@ -58,69 +58,70 @@
 .logoContainer {
   width: 54px;
     height: 54px;
-    padding: 5px;
     display: flex;
     align-items: center;
     justify-content: center;
     border-radius: 12px;
-    margin-bottom: 28px;
-    background-color: #8baa36;
-  
-    @media screen and (min-width: 768px) {
-      width: 68px;
-      height: 68px;
-      padding: 7px;
-      margin-bottom: 47px;
-      
+  background-color: var(--accent-color-darker);
+  }
+  @media screen and (min-width: 768px) {
+    .logoContainer {
+    width: 68px;
+    height: 68px;
     }
   }
+  .logo {
+    width: 100%;
+    height: 100%;
+    fill: #8baa36;
+    stroke: #fafafa;
+  }
 
 .content {
   position: relative;
   align-items: center;
   align-self: center;
   display: flex;
-  margin-top: 40px;
+  margin-top: 47px;
   width: 540px;
   max-width: 100%;
   flex-direction: column;
 
   @media (max-width: 991px) {
-    margin-top: 40px;
+    margin-top: 28px;
   }
 }
```

Comment:
Napraw całego css-a na poprawnego ponieważ to nie jest to sass i nie można nestować `@media` w klamrach jak w sass
To jest zwykły css 
```css
.logoContainer {
  width: 54px;
  height: 54px;
  display: flex;
  align-items: center;
  justify-content: center;
  border-radius: 12px;
  background-color: var(--accent-color-darker);
}
  @media screen and (min-width: 768px) {
    .logoContainer {
    width: 68px;
    height: 68px;
  }
}
.logo {
  width: 100%;
  height: 100%    
  fill: #8baa36;
  stroke: #fafafa;
}
``` 

tak poprawnie wygląda `@media` w css

Reply:
Od razu dodaj przyciski które już są